vendor/vendor.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const net = require('net');
+const socket = net.Socket();
+require('dotenv').config();
+let faker = require('faker');
+
+socket.connect({ port: 3000, host: 'localhost' }, () => {
+  console.log('TCP SERVER IS CONNECTED!!');
+});
+
+setInterval(() => {
+  let Store = faker.company.companyName();
+  let Id = faker.random.number();
+  let CustomerName = faker.name.firstName() + ' ' + faker.name.lastName();
+  let Address =
+    faker.address.streetAddress() +
+    ' ' +
+    faker.address.city() +
+    ' ' +
+    faker.address.state() +
+    ' ' +
+    faker.address.zipCode();
```

Comment:
Make this into an object because then all the parts are in one place. This will make it easier to add/remove proprieties if needed. Also you can just call the object instead calling each variable like on line 28..